src/style.css

```diff
@@ -0,0 +1,38 @@
+body {
+  background-color: black;
+  margin: 0;
+}
+
+.blue-box {
+  width: 350px;
+  background-color: #308dc0;
+  margin-bottom: 40px;
+  margin-left: 20px;
+  border-width: 10px;
+  border: 1px solid #e75551;
+  box-sizing: border-box;
+  padding: 10px;
+}
+
+.yellow-box {
+  background-color: #ffa500;
+  padding: 10px;
+  border: 1px solid #e75551;
```

Comment:
both this rules appears in three classes
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.

Reply:
understood